app/views/posts/index.html.erb

```diff
@@ -1,3 +1,18 @@
-<h1>Latest Posts</h1>
-<%= render 'list' %>
+<% if params[:search].present? %>
+  <% if @posts.size == 0 %>
+    <div class="alert alert-info">
+      <button type="button" class="close" data-dismiss="alert">&times;</button>
+      <span>No result found</span>
+    </div>
+    <h1>Latest Posts</h1>
+    <% @posts = Post.all %>
```

Comment:
Hi Andrew,
please advise where to place the code showing  the "Latest Posts" in case "No result found" -in the view file, or in the Posts Controller file. Our coach Andrew has advised me, that it will be better to be put in the controller file.
Thanks